modules/tinymce/src/core/test/ts/module/test/DialogUtils.ts

```diff
@@ -0,0 +1,11 @@
+import { Dialog } from '@ephox/bridge';
+
+import Editor from 'tinymce/core/api/Editor';
+import { WindowParams } from 'tinymce/core/api/WindowManager';
+
+const open = <T extends Dialog.DialogData>(editor: Editor, spec: Dialog.DialogSpec<T>, params: WindowParams): Dialog.DialogInstanceApi<T> =>
+  editor.windowManager.open(spec, params);
```

Comment:
Why was this one line of code refactored? It should be ok to just use `editor.windowManager.open` in the tests 🤔 

Or did you mean for `dummyDialog` to be in this utils file as well?

Reply:
I was import `searchreplace` for the tests 😛  I have move dummyDialog into the DialogUtils in core so that it can be reused.